doc/flame/examples/lib/flower.dart

```diff
@@ -50,7 +53,10 @@ class Flower extends PositionComponent with TapCallbacks {
   @override
   void render(Canvas canvas) {
     for (var i = 0; i < _paths.length; i++) {
-      canvas.drawPath(_paths[i], _paints[i]);
+      canvas.drawPath(
+        _paths[i],
+        getPaint(FlowerPaint.values.elementAt(i)),
```

Comment:
> This will be a good usage of the layers in #2072 later!

Assuming this gets merged soon 😅

Reply:
I'm a bit concerned how well all this is going to play with paintLayers. It seems only the main paint and paintIds are taken into account (understandably given paintLayers isn't merged yet!) but hopefully this will be soon (maybe?). I haven't looked at this PR till now myself but - what do you think?